.github/workflows/simulation-1.2.yml

```diff
@@ -98,3 +98,39 @@ jobs:
     - name: Codecov
       run: |
         bash <(curl -s https://codecov.io/bash) -x "${{ matrix.compiler.gcov }}" -Z
+
+  packet-verification:
+    runs-on: ubuntu-18.04
+    env:
+      REFERENCE_DEVICE: 1
+      VIRTUAL_TIME: 1
+      COVERAGE: 1
+      PACKET_VERIFICATION: 1
+      THREAD_VERSION: 1.2
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bootstrap
+        run: |
+          sudo rm /etc/apt/sources.list.d/* && sudo apt-get update
+          sudo apt-get --no-install-recommends install -y g++-multilib python3-setuptools python3-wheel ninja-build
+          python3 -m pip install -r tests/scripts/thread-cert/requirements.txt
+      - name: Build
+        run: |
+          ./script/test build
+      - name: Run
+        run: |
+          for i in {1..10}
+          do
+            ./script/test cert_suite ./tests/scripts/thread-cert/v1_2_LowPower*.py
+          done
+      - uses: actions/upload-artifact@v2
+        if: ${{ failure() }}
+        with:
+          name: packet-verification-pcaps
+          path: |
+            *.pcap
+            *.json
+      - name: Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          fail_ci_if_error: true
```

Comment:
If there are also 1.1 coverage, we need to upload coverage one at a time?

Reply:
Current 1.1 is not involved. And as `INTER_OP` is not set in this item, there won't be 1.1 directory. I would add it when there are cases having 1.1 device involved. However I would add bbr coverage here.